server/i18n-bootstrap/index.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import superagent from 'superagent';
+
+/**
+ * Internal dependencies
+ */
+const LANG_REVISION_FILE_URL = 'https://widgets.wp.com/languages/calypso/lang-revisions.json';
+
+export const fetchLangRevisions = () => {
+	return new Promise( ( resolve, reject ) => {
```

Comment:
Do we even need the `Promise` wrapper? Seems like the result from `superagent.get()` [should be `then`able](https://visionmedia.github.io/superagent/#request-basics), so I think I'd suggest dropping this file altogether, and directly using

```js
superagent.get( LANG_REVISION_FILE_URL )
	.then( /*...*/ )
	.catch( /*...*/ );
```

in `server/pages/index.js`.